recipes/pytest-instafail/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pytest-instafail" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 84c87dd708f00d248fb062cdfaf5ba14bf10ce68ce56d46d58f20aa882a33924
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pytest >=2.9
+
+test:
+  imports:
+    - pytest_instafail
+
+about:
+  home: https://github.com/pytest-dev/pytest-instafail
+  license: BSD
```

I think it should be 
```suggestion
  license: BSD-3-Clause
```